pages/integrations/bitbucket_server.md.erb

```diff
@@ -0,0 +1,33 @@
+# Bitbucket Server
+
+Buildkite integrates with Bitbucket Server to provide automated builds based on your source control. You can run a build every time you push code to BitBucket Server, using a webhook that you create in your Bitbucket Server.
+
+<div class="Docs__note">
+This guide shows you how to set up your BitBucket Server builds with Buildkite. It was written using Bitbucket server version 7.11.1
+</div>
+
+{:toc}
+
+## Step 1: connect Bitbucket Server and set up a pipeline
+
+1. In your *Settings*, navigate to *Repository Providers*.
```

Comment:
Style guide says we should not punctuate these.

Reply:
We should distinguish between person and org settings

![image](https://user-images.githubusercontent.com/95874/115543688-f348c900-a2a1-11eb-9432-70e9316107e0.png)